src/BlockchainWriter/ControllerConfiguration.ts

```diff
@@ -1,4 +1,4 @@
-export interface ClaimControllerConfiguration {
+export interface ControllerConfiguration {
```

Comment:
I've been wondering can we move this to the same class as the file? It belongs to the class anyway. I think it would make jumping between files easier and the imports easier. Doesn't need to be in this PR just something I've been meaning to ask.

Reply:
Yep! Sounds like a good idea.